config/core/resources/trigger.yaml

```diff
@@ -55,22 +55,22 @@ spec:
             properties:
               broker:
                 type: string
-                description: "Broker that this trigger receives events from. If not specified, will default to 'default'."
```

Comment:
" If not specified, will default to 'default'.", isn't it valid anymore?

Reply:
I removed this following changes from https://github.com/knative/eventing/pull/3573 (which I forgot to mention this PR supersedes). Let me know if you think I should add it back.

Reply:
I think this defaulting behavior exists and we should mention it in the description. Also, I see that Ville there leaves this https://github.com/knative/eventing/pull/3573/files#diff-35c872630bcc973ea91f0d72122bb2f7R60 untouched, and removes that from the `trigger_types.go`, but I'm not sure why.
WDYT?

Reply:
the default behavior exists in v1beta1, not v1. I would add it just in the v1beta1 schema.

Reply:
Yeah, you are right, I checked only `v1beta1`

Reply:
added it here for v1beta1 schema.